namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/02-limitrange.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limitrange
+  namespace: manage-soc-cases-preprod
+spec:
+  limits:
+  - default:
+      cpu: 1000m
+      memory: 2048Mi
+    defaultRequest:
+      cpu: 10m
+      memory: 1024Mi
```

Comment:
Please replace this with the standard default memory request of 100Mi

For more information, please see this page: https://user-guide.cloud-platform.service.justice.gov.uk/documentation/concepts/namespace-limits.html#namespace-container-resource-limits

Reply:
Thanks for the quick review turnaround. I've updated the request memory as suggested, and added slack-channel and team-name to annotations.